runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesPlugin.java

```diff
@@ -68,6 +82,8 @@
 	private static final String EDIT_NOTE = "Edit Note";
 	private static final String NOTE_PROMPT_FORMAT = "%s's Notes<br>" +
 		ColorUtil.prependColorTag("(Limit %s Characters)", new Color(0, 0, 170));
+	private static final int ICON_WIDTH = 14;
+	private static final int ICON_HEIGHT = 12;
```

Comment:
How do you come to 14x12 as the dimensions? The scripts seem to indicate that the "Previous name" icon is 14x14

Reply:
this is the dimensions of the icon that we add